homeassistant/components/yale_home/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "yale_home",
+  "name": "Yale Home",
+  "integration_type": "virtual",
+  "supported_by": "august"
```

Comment:
Will this integration show up under the August brand or under the Yale brand?

Reply:
neither at this point :)

I think this domain should be added to the `brands/yale.json` file.